### PR TITLE
Se 464 create a markdown file for the implementation of the minimum hold time rule

### PR DESCRIPTION
**Pull request by TJ-Everett**
Added getter function to ERC721 Handler for isMinHoldTimeActive()

### CODE REVIEW

File: docs/userGuides/rules/MINIMUM-HOLD-TIME.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Minimum Account Balance By Date Rule
+
+## Purpose
+
+The purpose of the minimum-hold-time rule is to reduce trade volitility by preventing transfers of tokens for a number of hours after ownership is acquired, either via minting or transfers. This rule allows developers to specifiy a number of hours, up to 43830 (5 years), that each tokenId must be held for.  
+
+## Applies To:
+
+- [ ] ERC20
```
Comment (oscarsernarosero) [inline docs/userGuides/rules/MINIMUM-HOLD-TIME.md:9]: There is a minBalByDate rule in the ERC20 handler as well. Is it a different kind?

**Review by oscarsernarosero — APPROVED** [review]
It looks really good. It only needs to check out the box for ERC20s

File: docs/userGuides/rules/MINIMUM-HOLD-TIME.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+# Minimum Account Balance By Date Rule
```
Comment (mpetersoCode55) [inline docs/userGuides/rules/MINIMUM-HOLD-TIME.md]: Shouldn't this say Minimum Hold Time Rule?

File: docs/userGuides/rules/MINIMUM-HOLD-TIME.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+## Data Structure
+
+The rule is a uint32 variable for number of hours each individual token must be held by the owner of that tokenId, up to a maximum of 43830 hours or 5 years. 
+
+```c
+/// simple rule(with single parameter) variables
+    uint32 private minimumHoldTimeHours;
+```
+
+Additionally, each starting unix timestamp for the ownership of the tokenId is stored in a mapping inside the hanlder.
```
Comment (mpetersoCode55) [inline docs/userGuides/rules/MINIMUM-HOLD-TIME.md]: sp on handler

**Review by ShaneDuncan602 — APPROVED** [review]
Looks great.

**Pull request merged**